### PR TITLE
add missing deps and fix cli error

### PR DESCRIPTION
**Pull request by dineshyv**

### CODE REVIEW

File: src/llama_stack_client/lib/cli/llama_stack_client.py [hunk 1]
```diff
@@ -34,14 +34,12 @@ def cli(ctx, endpoint: str, config: str | None):
     ctx.ensure_object(dict)
 
     # If no config provided, check default location
-    if config is None:
+    if config:
```
Comment (raghotham) [inline src/llama_stack_client/lib/cli/llama_stack_client.py:37]: Is this change correct?

File: pyproject.toml [hunk 2]
```diff
@@ -15,6 +15,12 @@ dependencies = [
     "distro>=1.7.0, <2",
     "sniffio",
     "cached-property; python_version < '3.8'",
```
Comment (raghotham) [inline pyproject.toml:17]: Can’t we just force python 3.10+

File: src/llama_stack_client/lib/cli/llama_stack_client.py (hunk 1 above)
Comment (dineshyv) [reply src/llama_stack_client/lib/cli/llama_stack_client.py:37]: yes, if config is provided, only then we need to enter this condition.

Comment (dineshyv) [reply src/llama_stack_client/lib/cli/llama_stack_client.py]: thanks for catching this. fixed.

File: src/llama_stack_client/lib/cli/llama_stack_client.py [hunk 3]
```diff
@@ -34,14 +34,12 @@ def cli(ctx, endpoint: str, config: str | None):
...
         if endpoint != "":
             raise ValueError("Cannot use both config and endpoint")
         default_config = get_config_file_path()
         if default_config.exists():
```
Comment (yanxi0830) [inline src/llama_stack_client/lib/cli/llama_stack_client.py:42]: Do we want to override original config when using `--config`?

Comment (dineshyv) [reply src/llama_stack_client/lib/cli/llama_stack_client.py:42]: ya, fixed it.

File: pyproject.toml (hunk 2 above)
Comment (dineshyv) [reply pyproject.toml:17]: I dont think we force this on client side currently.

**Pull request merged**